R/build_lm.R

```diff
@@ -871,6 +931,17 @@ xlevels_to_base_level_table <- function(xlevels) {
   ret
 }
 
+vif_to_dataframe <- function(x) {
+  ret <- NULL
+  if (is.matrix(x$vif)) {
+    ret <- x$vif %>% as.data.frame() %>%  tibble::rownames_to_column(var="term") %>% rename(VIF=GVIF)
+  }
+  else {
+    ret <- data.frame(term=names(x$vif), VIF=x$vif)
+  }
+  ret
+}
+
 #' special version of tidy.lm function to use with build_lm.fast.
 #' @export
```

Comment:
Should describe that it returns an empty data frame in case of error.

Reply:
Added comment.